superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -131,21 +134,11 @@ def get_recipe_text(self, distributor, license_text, die_msg=None):
         self.get_license_line()
         if isinstance(self.license, str):
             self.license = self.license.split(',')[0]
-            self.license = self.license.replace(' ', '-')
-            ret += 'LICENSE = "' + self.license + '"\n'
+            ret += 'LICENSE = "'
+            ret += ' & '.join([get_license(l) for l in self.license]) + '"\n'
         elif isinstance(self.license, list):
```

Comment:
So I don't think you need to use a join when `self.license` is a string. I think this line could just be

```py
ret += 'LICENSE = "%s"\n' % getlicense(self.license)
```

Reply:
Ahhh, i didnt see that '[0]' on line 136, thought it was just being made into a list there so i treated it the same, my b!~